Sources/Retired.swift

```diff
@@ -10,25 +10,25 @@ public typealias RetiredCompletion = (Bool, Message?, NSError?) -> Void
 
 public let RetiredErrorDomain = "RetiredError"
 
-public enum RetiredError: ErrorType {
-  case NotConfigured
+public enum RetiredError: Error {
+  case notConfigured
 }
 
-public class Retired {
+open class Retired {
```

Comment:
this is now subclassable outside the module (was chosen by the convert-a-tron -- you care either way?

https://github.com/apple/swift-evolution/blob/master/proposals/0117-non-public-subclassable-by-default.md